.github/workflows/pytest.yml

```diff
@@ -0,0 +1,20 @@
+name: PyTest
+on: pull_request
+jobs:
+  pytest:
+    name: Run the application's tests with PyTest
+    runs-on: ubuntu-20.04
+    container: fedora:34
+    steps:
+      - name: Install Pipenv and git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Setup environment
+        run: pipenv sync --dev
+      - name: Run PyTest
+        run: pipenv run pytest -v
```

Comment:
Apparently it's not important to this kind of files.
Just like Flake8, it works without a new line in the end of the file.

Reply:
Note there is a long discussion about adding EOL (newline character) at EOF on various platforms,
not adding new line at the end of a file may result in unexpected behavior of different tools (e.g: ignoring the last line of the file).
Also most text editors will add it automatically, so next time this file is edited it will be part of the diff.
Moreover it is not consistent with most of the files in the project.